### PR TITLE
IDE-4822 IDE support both of blade 3.9.2 and 4.0.0 version.

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
ci:retest

File: tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java [hunk 1]
```diff
@@ -555,7 +591,7 @@ private void _loadGradleWorkspaceProperties() {
 
 			IPath gradleProperties = projectLocation.append("gradle.properties");
 
-			properties = PropertiesUtil.loadProperties(gradleProperties);
+			properties.putAll(PropertiesUtil.loadProperties(gradleProperties));
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java:594]: Do we need to clear the previous properties?  What if a user changes their properties?

File: tools/plugins/com.liferay.ide.project.core/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-lib/blade.jar
\ No newline at end of file
+lib/blade.jar
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/.gitignore:1]: Lets put the versions on these blade-4.0.0 and blade-3.9.2

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java [hunk 3]
```diff
@@ -14,148 +14,50 @@
...
 
 /**
  * @author Gregory Amerson
  * @author Andy Wu
+ * @author Simon Jiang
  */
 public class BladeCLI {
 
-	public static final String BLADE_CLI_REPO_URL = "BLADE_CLI_REPO_URL";
+	public static final String NEW_BLADE_JAR_FILE_NAME = "blade.jar";
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java:49]: Lets use BLADE_392 and BLADE_400

File: tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java (hunk 1 above)
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java:594]: we still need to read build type from .balde.properties. I changed it to pullAll just want to keep content .blade.properties.

**Comment by simonjhy** [comment]
ci:retest

**Pull request closed without merging**